### ISSUE TITLE
Show total runtime of media

### ISSUE THREAD

**Issue by pete2583** [Tautulli/Tautulli#2141]
### What is your feature request?

It would be nice to list the total runtime of all media and total library size in tb similar to how Plex dash displays it. 

It would nice to have it amongst the library total files. 

Here's an image from plex dash incase you don't know how it looks. https://i.postimg.cc/NjKKWfbM/received-738267144774203.jpg

**Comment by herby2212** [comment]
The total runtime of a library is already visible as `Global Stats` when selecting a library.
The total size in TB is currently not visible, but you could get it by using the exporter configured like so:
![image](https://github.com/Tautulli/Tautulli/assets/12448284/7180f990-dd3b-40c1-9060-48d8638c3c46)

The size could also be integrated as a stat or in something like this: https://github.com/Tautulli/Tautulli/pull/1472

**Comment by jaypi95** [comment]
I don't think the time displayed in the global stats is the total runtime of the media but rather the total time it has been played so far.

**Comment by herby2212** [comment]
> I don't think the time displayed in the global stats is the total runtime of the media but rather the total time it has been played so far.

Agreed. I misunderstood the requirement there the watch time actually displays the time the media has really been watched by users as you mentioned correctly.

To get the runtime the exported again could be used by using `media.duration` in `Custom Media Info Fields`.

A possible solution would be the addition of those two stats in the `Media Info` section of the library.
